routers/repo/issue.go

```diff
@@ -1346,10 +1385,41 @@ func UpdateCommentContent(ctx *context.Context) {
 		return
 	}
 
+	files := ctx.QueryStrings("files[]")
+	for i := 0; i < len(comment.Attachments); i++ {
+		if !util.IsStringInSlice(comment.Attachments[i].UUID, files) {
```

Comment:
Same here

Reply:
Update, @lafriks thank you fro the review. 😄